src/components/CampaignInitialisation/UpdateFromAppStoreContent.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FunctionComponent } from "react";
+import { AppText } from "../Layout/AppText";
+import { sharedStyles } from "./sharedStyles";
+import { View, Platform, Linking } from "react-native";
+import { DarkButton } from "../Layout/Buttons/DarkButton";
+import { Feather } from "@expo/vector-icons";
+import { size, color } from "../../common/styles";
+
+export const UpdateFromAppStoreContent: FunctionComponent = () => {
+  const supportLink = `mailto:supplyallyhelp@hive.gov.sg?subject=[SupplyAlly App Store Update Error]`;
+  let storeName: string;
+  let storeLink: string;
+  switch (Platform.OS) {
+    case "ios":
+      storeName = "App Store";
+      storeLink = "https://apps.apple.com/sg/app/supplyally/id1497126533";
+      break;
+
+    case "android":
+    default:
+      // figure out a better way to handle default
```

Comment:
If this is a TODO, might be useful to add the prefix so we can find this easily~

Reply:
Good catch!!